test/unit/repositories/test_attachment.py

```diff
@@ -86,3 +91,68 @@ def test_create(self):
         self.mock_create(ATTACHMENT_IN_DATA_ALL_VALUES)
         self.call_create()
         self.check_create_success()
+
+
+class ListDSL(AttachmentRepoDSL):
+    """Base class for `list` tests."""
+
+    _expected_attachment_out: list[AttachmentOut]
+    _entity_id_filter: Optional[str]
+    _obtained_attachment_out: list[AttachmentOut]
+
+    def mock_list(self, attachment_in_data: list[dict]) -> None:
+        """
+        Mocks database methods appropriately to test the `list` repo method.
+
+        :param attachment_in_data: List of dictionaries containing the attachment data as would be required for an
+            `AttachmentIn` database model (i.e. no ID or created and modified times required).
+        """
```

Comment:
```suggestion
            `AttachmentIn` database model (i.e. no created and modified times required).
        """
```
`AttachmentIn` models actually do contain the ID already. There's a similar error in the `test/unit/repositories/test_image.py` file if you could fix that as well